### ISSUE TITLE
Link list does not respect "Open external links in new tabs = true" setting

### ISSUE THREAD

**Issue by WalterFaber** [Kovah/LinkAce#339]
LinkAce 1.7.0
Situation: Link list does not respect "Open external links in new tabs = true" setting

Steps to reproduce:
1. Set "Open external links in new tabs" = true in settings
2. Navigate to "All Links"
2. Click on any link

Expectation: link opens up in a new tab
Actual: link opens up in the current tab

Great app, thank you a lot Kevin and all other contributors!

**Comment by Kovah** [comment]
Hi WalterFaber!
I have tested this both with version 1.7 and 1.6, both work correctly. Is it possible that your browser somehow blocks this setting?
You could try two things:
- disable the setting and enable it again, maybe it was not saved properly
- check if the links should open in new tabs: right click any link and select "inspect element" (or similar, depending on your browser), then check if the link is displayed like this in the window which pops up: `<a href="https://some-link.com" target="_blank">...`. If that's the case it seems to be your browser.

**Comment by WalterFaber** [comment]
Hello Kevin,

thank you for the quick response.

Sorry, I should have been more precise in my description.
I use FF 95.0.18
"New tab" works for the same link in other views (e.g. within lists)
I experience it only in view "All links" AND if I have selected "list with few details" in the settings.
If I select "Cards with less dails", the setting works correctly.
"List with many details" doesn't work at the moment, so I can't test it (different issue - I guess caching related)

I also checked the target attribute in the a href- tag... it is missing in this particular view but available in the other views.

This is how it look likle in the affected view:
`<a href="https://www.linkace.org/" title="https://www.linkace.org/">`

And this is the same link within the list view:
`<a href="https://www.linkace.org/" target="_blank" rel="noopener noreferrer">`
